buffer/src/main/java/io/netty/buffer/ByteBufUtil.java

```diff
@@ -706,6 +706,72 @@ public static void copy(AsciiString src, int srcIdx, ByteBuf dst, int length) {
     }
 
     /**
+     * Creates a retained duplicate of the specified {@link ByteBuf}.
+     *
+     * @see ByteBuf#retainedDuplicate()
+     */
+    public static ByteBuf retainedDuplicate(ByteBuf buf) {
```

Comment:
now that we are only doing this on pooled buffers .. and explicitly use the pooled types in this method ... should we make this package private?